{"name": "test_login_with_invalid_user", "status": "passed", "start": 1751995241876, "stop": 1751995244496, "uuid": "bf37c7dd-3925-4c59-b61a-51d7f829508e", "historyId": "0e4edb4712d043b1bc1ffb1b2129d6cf", "testCaseId": "0e4edb4712d043b1bc1ffb1b2129d6cf", "fullName": "testcase.test_login.TestLogin#test_login_with_invalid_user", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-CIEP9G2C"}, {"name": "thread", "value": "10920-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}]}